lib/bump/pull_request_creator.rb

```diff
@@ -80,35 +81,46 @@ def pr_name
     end
 
     def pr_message
-      if dependency.github_repo_url
-        msg = "Bumps [#{dependency.name}](#{dependency.github_repo_url}) to "\
+      if dependency_metadata_finder.github_repo_url
+        msg = "Bumps [#{dependency.name}]"\
+              "(#{dependency_metadata_finder.github_repo_url}) to "\
               "#{dependency.version}."
       else
         msg = "Bumps #{dependency.name} to #{dependency.version}."
       end
 
-      if dependency.changelog_url
-        msg += "\n- [Changelog](#{dependency.changelog_url})"
+      if dependency_metadata_finder.changelog_url
+        msg += "\n- [Changelog](#{dependency_metadata_finder.changelog_url})"
       end
 
-      if dependency.github_repo_url
+      if dependency_metadata_finder.github_repo_url
         title = if dependency.previous_version
                   "Changes since #{dependency.previous_version}"
                 else
                   "Commits"
                 end
-        msg += "\n- [#{title}](#{dependency.github_compare_url})"
+        msg += "\n- [#{title}]"\
+               "(#{dependency_metadata_finder.github_compare_url})"
       end
 
       msg
     end
 
     def default_branch
-      @default_branch ||= Github.client.repository(watched_repo).default_branch
+      @default_branch ||= github_client.repository(watched_repo).default_branch
     end
 
     def new_branch_name
       "bump_#{dependency.name}_to_#{dependency.version}"
     end
+
+    def dependency_metadata_finder
+      @dependency_metadata_finder ||=
+        begin
+          finder_class =
+            DependencyMetadataFinders.for_language(dependency.language)
+          finder_class.new(dependency: dependency, github_client: github_client)
+        end
```

Comment:
It's a hangover from copying this method across from `Dependency`, before we moved to having a `for_language` method on the module. I'll update now - would be awesome to get this merged :)